website/docs/r/rde_type.html.markdown

```diff
@@ -0,0 +1,107 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_rde_type"
+sidebar_current: "docs-vcd-resource-rde-type"
+description: |-
+   Provides the capability of creating, updating, and deleting Runtime Defined Entity types in VMware Cloud Director.
+---
+
+# vcd\_rde\_type
+
+Provides the capability of creating, updating, and deleting Runtime Defined Entity types in VMware Cloud Director.
+Requires system administrator privileges.
+
+Supported in provider *v3.9+*
+
+## Example Usage with a schema file
+
+```hcl
+data "vcd_rde_interface" "my_interface" {
+  vendor  = "bigcorp"
+  nss     = "tech1"
+  version = "1.2.3"
+}
+
+resource "vcd_rde_type" "my_rde_type" {
+  vendor        = "vmware"
+  nss           = "vcd"
+  version       = "4.5.6"
+  name          = "My VMware RDE Type"
+  interface_ids = [data.vcd_rde_interface.my_interface.id]
+  schema        = file("${path.module}/schemas/my-type-schema.json")
+}
+```
+
+## Example Usage with a URL that contains a schema file
+
+```hcl
+data "vcd_rde_interface" "my_interface" {
+  vendor  = "bigcorp"
+  ns      = "tech1"
+  version = "1.2.3"
+}
+
+resource "vcd_rde_type" "my_rde_type" {
+  vendor        = "vmware"
+  nss           = "vcd"
+  version       = "4.5.6"
+  name          = "My VMware RDE Type"
+  interface_ids = [data.vcd_rde_interface.my_interface.id]
+  schema_url    = "https://just.an-example.com/schemas/my-type-schema.json"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `vendor` - (Required) The vendor of the Runtime Defined Entity Type.
+* `nss` - (Required) A unique namespace associated with the Runtime Defined Entity Type.
+* `version` - (Required) The version of the Runtime Defined Entity Type. Must follow [semantic versioning](https://semver.org/) syntax.
+* `name` - (Required) The name of the Runtime Defined Entity Type.
+* `description` - (Optional) The description of the Runtime Defined Entity Type.
+* `interface_ids` - (Optional) The set of [Defined Interfaces](/providers/vmware/vcd/latest/docs/resources/rde_interface) that this Runtime Defined Entity Type will use.
+* `schema` - (Optional) A string that specifies a valid JSON schema. It can be retrieved with functions such as `file`, `templatefile`... Either `schema` or `schema_url` is required.
```

Comment:
What is the thought behind having the three dots here: "`templatefile`..."?

Reply:
Maybe I should put `etc` instead.

Reply:
Thanks. Maybe also "with functions" -> "with Terraform functions"?

Reply:
Done!